components/resources/ai_chat_prompts.grdp

```diff
@@ -39,6 +39,63 @@ Here is the conversational history (between the user and you) prior to the quest
 <message name="IDS_AI_CHAT_QUESTION_PROMPT_SEGMENT" translateable="false">
 Propose up to 3 very short questions that a reader may ask about the content. Consider intriguing or unusual elements of the content, or structurally important. Separate each suggested question with the character "|". End the question list with a &lt;/response&gt; tag.</message>
 
-<message name="IDS_AI_CHAT_QUESTION_SUMMARIZE_VIDEO" translateable="false">Provide a concise bullet list of the most important points of the video along with a short summary of each point. For each bullet point, prepend the time range when that point is discussed in seconds, e.g. [23s-68s].</message>
+<message name="IDS_AI_CHAT_QUESTION_SUMMARIZE_VIDEO_TIMESTAMPS" translateable="false"> For each bullet point, prepend the time range when that point is discussed in seconds, e.g. [23s-68s].</message>
+
+<message name="IDS_AI_CHAT_QUESTION_SUMMARIZE_VIDEO_BULLETS" translateable="false">Provide a concise bullet list of the most important points of the video along with a short summary of each point.</message>
+
+<message name="IDS_AI_CHAT_LLAMA2_SYSTEM_MESSAGE_GENERIC" translateable="false">
```

Comment:
Could we extract the system message into a single string? Likely it will be the same across tasks as it's used to set the chat's tone and limits/scope

Reply:
It's set up now such that the article / video context is included in the system prompt.  That way the first message "user message" is from the user.  Alternatively we could use a generic / uniform system prompt across everything tat, and hijack the first user prompt to inject the article / video context.

Reply:
Also I'm wondering if we can get a more optimized system message for the prompt that generates questions that focuses on being concise and returning structured data. Do we need the model to assume the role of Leo for that exchange?

Reply:
The system message is intended to set the chat's tone and constraint across turns, it's were we would specify the assistant to _act as_ a role and the rules that it should follow. The article/video context should be part of the user first request as that's what the request is related to. That's why I see it more like a separate string (the chatbot character should be the same across applications, or at least the type of applications we are considering now)